.github/scripts/detect_branch.sh

```diff
@@ -1,6 +1,6 @@
 case $GITHUB_EVENT_NAME in
-  pull_request)          source_branch="${GITHUB_HEAD_REF#refs/heads/}" ;;
```

Comment:
I keep seeing whitespace changes on these lines.  Are you using different editors or different editor configs?

In the `elastio` repo Vitaliy recently added a `.editorconfig` file which most editors will respect, to avoid this oscillation.  You should consider that for this repo as well.

Reply:
I use a single editor. the white-spaces were corrected according to the code shell scripts stile (4 spaces per tab). Before that it was incorrect as result of copy-paste from the ci.yml with the 2 spaces ber tab according to rule for the yaml files.

> In the elastio repo Vitaliy recently added a .editorconfig file which most editors will respect, to avoid this oscillation. You should consider that for this repo as well.

it's not necessary but thanks for letting me know.